### PR TITLE
Update composer docs

### PR DESCRIPTION
**Pull request by exussum12**
OSX issue has been fixed for later versions

### CODE REVIEW

**Comment by exussum12** [comment]
@alcohol not sure who reviews PR's here ? The docker for mac issue has been recently resolved

File: composer/content.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ $ eval $(ssh-agent); \
   %%IMAGE%% install
 ```
 
-**Note:** This currently does not work on OSX, see [docker/for-mac#410](https://github.com/docker/for-mac/issues/410).
+**Note:** This on OSX this requires Docker For Mac v2.2.0.0 or later, see [docker/for-mac#410](https://github.com/docker/for-mac/issues/410).
```
Comment (alcohol) [inline composer/content.md]:
```suggestion
**Note:** On OSX this requires Docker For Mac v2.2.0.0 or later, see [docker/for-mac#410](https://github.com/docker/for-mac/issues/410).
```

**Comment by alcohol** [comment]
Aside from the suggested change, I personally do not recall either how PRs work on this repo.

File: composer/content.md (hunk 1 above)
Comment (alcohol) [reply composer/content.md]: If you can adjust this, I can approve the PR.

**Comment by tianon** [comment]
Doh, sorry!  Thanks for getting this fixed up. 👍

Typically we try to ping the image maintainers to verify changes here before we merge things, this one just slipped through the cracks. 😅

**Pull request merged**